osaca/parser/immediate.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+from osaca.parser.operand import Operand
+
+
+class ImmediateOperand(Operand):
+    def __init__(
+        self,
+        identifier_id=None,
```

Comment:
Same as in `InstructionForm` class, change
  - `identifier_id` to `identifier`
  - `type_id` to `imd_type` (`type` is already taken)
  - `value_id` to `value`
  - `shift_id` to `shift`

Reply:
[46004add41e8f4669dd4c6041f2ff98ef0cac932](https://github.com/RRZE-HPC/OSACA/pull/102/commits/46004add41e8f4669dd4c6041f2ff98ef0cac932)